src/sentry/eventstore/base.py

```diff
@@ -66,6 +66,18 @@ class Columns(Enum):
     CONTEXTS_VALUE = "contexts.value"
 
 
+class Filter(object):
```

Comment:
plaease let's add a docstring to this class and some notes to the expected types for the constructor parameters.

Reply:
Added